admin/sql/create_tables.sql

```diff
@@ -4,7 +4,8 @@ CREATE TABLE "user" (
   id             SERIAL,
   created        TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
   musicbrainz_id VARCHAR NOT NULL,
-  auth_token     VARCHAR
+  auth_token     VARCHAR,
+  last_login     TIMESTAMP WITH TIME ZONE
```

Comment:
This value can be null. What does a null last login mean? Will you check this correctly every time that you check a user's last login?
Perhaps a better value for this would be `created` for all existing users, then alter the column to make it `NOT NULL` (in the update - perhaps since this has been merged you have another update script to set this)